roles/network_plugin/multus/defaults/main.yml

```diff
@@ -7,3 +7,4 @@ multus_cni_conf_dir: "{{ ('/host', multus_cni_conf_dir_host) | join }}"
 multus_cni_bin_dir: "{{ ('/host', multus_cni_bin_dir_host) | join }}"
 multus_cni_run_dir: "{{ ('/host', multus_cni_run_dir_host) | join }}"
 multus_kubeconfig_file_host: "{{ (multus_cni_conf_dir_host, '/multus.d/multus.kubeconfig') | join }}"
+multus_namespace_isolation: true
```

Comment:
According to your provided [docs](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/configuration.md), this default value is `false` not `true`.

Reply:
Indeed. It's weird because I've actually tested and without forcing the flag to `false` my network config placed in another NS was not taken in account.
I looked into the Mutlus code. The bool is created but never set so by default in Golang it should be false.
I've updated the default var and the doc accordingly in my PR.